trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/probability-of-default.njk

```diff
@@ -5,6 +5,8 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle %}What’s the probability of default for {{ deal.submissionDetails.supplierName}}{% endblock %}
```

Comment:
Should we have a `?` at the end of the title?

Reply:
Thank you, I added `?`